src/sap/fhir/model/r4/FHIRUtils.js

```diff
@@ -497,14 +503,61 @@ sap.ui.define([
 		} else if (oFilter.sOperator === FHIRFilterOperator.BT) {
 			var oValue2 = FHIRFilterOperatorUtils.getFilterValue(oFilter.oValue2);
 			mParameters[sPath + sFhirSearchModifier] = ["ge" + oValue1, "le" + oValue2];
-		} else if (bLogicalConnection && !bLogicalOperator){
-			if (mParameters[sPath + sFhirSearchModifier]){
+		} else if (bLogicalConnection && !bLogicalOperator) {
+			if (mParameters[sPath + sFhirSearchModifier]) {
 				mParameters[sPath + sFhirSearchModifier].push(vValue);
 			} else {
-				mParameters[sPath + sFhirSearchModifier] = [ vValue ];
+				mParameters[sPath + sFhirSearchModifier] = [vValue];
 			}
 		} else {
-			mParameters[sPath + sFhirSearchModifier] =  vValue;
+			mParameters[sPath + sFhirSearchModifier] = vValue;
+		}
+	};
+
+	/**
+	 * Creates a complex filter
+	 *
+	 * @param {object} oFilter The filter which should be added to the parameters
+	 * @param {object} mParameters The parameters which should be passed to the request
+	 * @param {string} sLogicalConnection if the list of filters needs to be combined either with AND or OR
```

Comment:
its a optional parameter, so we can mark the parameter `@param {string} [sLogicalConnection]` also as optional

Reply:
done